README.md

```diff
@@ -11,6 +11,21 @@ These are basic Python based tools for working with medical images and text, spe
 The cleaning method used here mirrors the one by CTP in that we can identify images based on known
 locations. We are looking for collaborators to develop and validate an OCR cleaning method! Please reach out if you would like to help work on this.
 
+### New: Enhanced Private Tag Syntax
```

Comment:
These changes (docs) don't go in the README, they should go in the web docs (the pages under docs).

Reply:
Thanks @Simlomb. Do you have a second pair of eyes in mind to review?